docs/getting-started/testing.mdx

```diff
@@ -4,7 +4,7 @@ icon: 'vial'
 title: 'Testing & Validation'
 ---
 
-Testing is critical process when building and maintaining an authorization system. This page explains how to ensure the new authorization model and related authorization data works as expected in Permify.
+Testing is a critical process when building and maintaining an authorization system. This page explains how to ensure the new authorization model and related authorization data works as expected in Permify.
```

Comment:
```suggestion
Testing is a critical process when building and maintaining an authorization system. This page explains how to ensure your authorization model and data works as expected in Permify.
```
[suggestion]: reduce duplication of 'authorization' in this paragraph. 3 times is probably too many in such a small paragraph!